File: lte.c
#include "stdio.h"

int main()
{
   // Default to stdin
   FILE* ifp = stdin;
   int i, n, j, f;

   // Reads the number of test cases.
   fscanf(ifp, "%d", &n);

   // Loops through each test case.
   for (i = 1; i <= n; i++)
   {
      // Reads the two input integers.
      fscanf(ifp, "%d %d", &j, &f);
      // Prints the scenario number
      printf("Scenario #%d: ", i);
      // If the time to get driving directions is less than or equal to the time for Frohan to escape, then Frohan will be nabbed.
      if (j <= f)
         printf("Agent Jimmie Flowers nabs Frohan!\n");
      // Else he gets away.
      else
         printf("Missed it by that much.\n");
   }

   return 0;
}

File: lte.java
import java.util.*;
import java.io.*;
public class lte {
   public static void main(String[] args)throws IOException{
      Scanner br = new Scanner(System.in);
      
      //Read in the number of test cases
      int t = br.nextInt();
      
      for(int c = 1;c<=t;c++){
         //Read in the two times
         int t1 = br.nextInt();
         int t2 = br.nextInt();
         
         //Agent Jimmie finishes first
         if(t1 <= t2){
            System.out.println("Scenario #"+c+": Agent Jimmie Flowers nabs Frohan!");
         }
         //Frohan escapes
         else{
            System.out.println("Scenario #"+c+": Missed it by that much.");
         }
      }
   }
}

